stores/hostdb.go

```diff
@@ -405,12 +405,18 @@ func (e *dbBlocklistEntry) BeforeCreate(tx *gorm.DB) (err error) {
 }
 
 func (e *dbBlocklistEntry) blocks(h dbHost) bool {
+	values := []string{e.Entry}
```

Comment:
Isn't this always true? You create `values := []string{e.Entry}` and then compare to `e.Entry` later.

Reply:
woops, should be `h.NetAddress`